app/core/less/containers.less

```diff
@@ -40,15 +41,8 @@
   }
 }
 
-.container-nav-inner {
-  padding-left: ~"calc((100% - 275px) / 2)";
-  padding-right: ~"calc((100% - 275px) / 2)";
-
-  @media(min-width: @screen-lg-min) {
-    padding-left: 0;
-    padding-right: 0;
-  }
-}
+// .container-nav-inner {
+// }
```

Comment:
still used in footerlink.js

Reply:
I kept this entry comment if we need to set something in it after.
I will remove the class in the footer.